easybuild/tools/options.py

```diff
@@ -288,6 +288,7 @@ def override_options(self):
                           "(e.g. --hide-deps=zlib,ncurses)", 'strlist', 'extend', None),
             'hide-toolchains': ("Comma separated list of toolchains that you want automatically hidden, "
                                 "(e.g. --hide-toolchains=GCCcore)", 'strlist', 'extend', None),
+            'latest-self': ("Install latest known version of easybuild", None, 'store_true', False),
```

Comment:
hmm, somehow my last comment on this got trashed...

let's go with `--install-latest-eb-release` here